docs/source/references/writing-connectors.md

```diff
@@ -0,0 +1,396 @@
+## Overview

```

Comment:
The footer on this page that is used to move to previous and next sections looks different to other pages (at least it does on my built version